### PR TITLE
Fix taxonomy page styles

### PR DESCRIPTION
**Pull request by hamrant**
Jira - https://propeople-us.atlassian.net/browse/OYPD-183

Steps for review:
- [x] login as admin
- [x] go to /blog/start-training-now-y-run
- [x] click on Category: "Community Outreach" link -  (/taxonomy/term/17)
- [x] check that that blog teasers looks fine (not beautiful but fine)
- [x] profit

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build331

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/331/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build331/profiles/openy/build/reports/behat

**Comment by alexschedrov** [comment]
@hamrat I think it would be better to do following things:
- Override taxonomy page template for this vocabulary and add markup for grid(layout should be as on one column page)
- Remove "Subscribe to Community Outreach"
- Would be nice to add pathauto pattern to generate good URL aliases for this vocabulary

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build335

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/335/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build335/profiles/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build336

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/336/

File: config/install/system.site.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Site-Install
```
Comment (alexschedrov) [inline config/install/system.site.yml]: Let's change to "Open Y Distribution"

**Review by alexschedrov — CHANGES_REQUESTED** [review]

File: modules/openy_features/openy_taxonomy/config/install/views.view.taxonomy_term.yml [hunk 2]
```diff
@@ -0,0 +1,313 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.teaser
+  module:
+    - node
+    - taxonomy
+    - user
+_core:
```
Comment (alexschedrov) [inline modules/openy_features/openy_taxonomy/config/install/views.view.taxonomy_term.yml]: Please get rid of hash.

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build336/profiles/openy/build/reports/behat

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build337

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/337/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build337/profiles/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build338

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/338/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build338/profiles/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build339

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/339/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build339/profiles/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/347/

**Build Log**
*last 25 lines*

 ```
[...truncated 203 lines...]
stdout: You are about to create a /var/www/build347/sites/default/settings.php file and CREATE the 'drupal347' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=12   changed=5    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build348

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/348/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build348/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**